contracts/Registry.sol

```diff
@@ -265,15 +261,13 @@ contract Registry is IRegistry, ERC165 {
         emit ServiceCreated(orgName, serviceName, orgName, serviceName);
```

Comment:
What do you say about adding the metadataURI as part of the event so that whenever there is a change we can trace it back.